docs/source/tutorials/otx_cli/how_to_train/detection.rst

```diff
@@ -0,0 +1,389 @@
+    }
+  }
+
+``Ann-files`` contains a path to the annotation., while ``data-roots`` is a path to the folder, where images are stored.
+
+*********
+Training
+*********
+
+1. First of all, we need to choose which object detection model will we train. The list of supported templates for object detection is available with the command line below. 
+
+.. note::
+
+  The characteristics and detailed comparison of the models could be found in :doc:`Explanation section <../../../explanation/Main_algorithms/object_detection>`.
+
+  To modify the architecture of supported models with various backbones, please refer to the :doc:`advanced tutorial for model customization <../../advanced/backbones>`.
+
+.. code-block::
+
+  (detection) ...$ otx find --template --task DETECTION
+  +-----------+-----------------------------------+-------+---------------------------------------------------------------------------+
+  |    TASK   |                 ID                |  NAME |                                    PATH                                   |
+  +-----------+-----------------------------------+-------+---------------------------------------------------------------------------+
+  | DETECTION |   Custom_Object_Detection_YOLOX   | YOLOX | otx/algorithms/detection/configs/detection/cspdarknet_yolox/template.yaml |
+  | DETECTION |  Custom_Object_Detection_Gen3_SSD |  SSD  |  otx/algorithms/detection/configs/detection/mobilenetv2_ssd/template.yaml |
+  | DETECTION | Custom_Object_Detection_Gen3_ATSS |  ATSS | otx/algorithms/detection/configs/detection/mobilenetv2_atss/template.yaml |
+  +-----------+-----------------------------------+-------+---------------------------------------------------------------------------+
+
+2. ``otx train`` trains a model (a particular model template) on a dataset and results in two files:
+
+- ``weights.pth`` - a model snapshot
+- ``label_schema.json`` - a label schema used in training, created from a dataset
+
+These are needed as inputs for the further commands: ``export``, ``eval``,  ``optimize``,  ``deploy`` and ``demo``.
+
+
+3. To have a specific example in this tutorial, all commands will be run on the ATSS model. It's a medium model, that achieves relatively high accuracy, while keeping the inference fast.
+
+The following command line starts 1 GPU training of the medium object detection model on BCCD dataset:
+
+.. code-block::
+
+  (detection) ...$ otx train otx/algorithms/detection/configs/detection/mobilenetv2_atss/template.yaml
+                            --train-ann-files data/wgisd/coco_annotations/train_bbox_instances.json 
+                            --train-data-roots  data/wgisd/data
+                            --val-ann-files data/wgisd/coco_annotations/test_bbox_instances.json 
+                            --val-data-roots data/wgisd/data 
+                            --save-model-to outputs
+                            --work-dir outputs/logs
```

Comment:
Do we need both? (sorry, I'm not sure)
@harimkang Could you advise on this?

Reply:
`save-model-to` saves the final snapshot and label_schema. While `work-dir` saves all training logs and files for tensorbord, what can be useful for a deep research. I'll explain more about it in tutorial, thank you